### PR TITLE
issue-27: enable jacoco and codecoverage

### PR DESCRIPTION
**Pull request by ww2lin**

### CODE REVIEW

File: feature-adapter-group/build.gradle [hunk 1]
```diff
@@ -40,3 +45,12 @@ dependencies {
     testImplementation deps.junit
     testImplementation deps.easymock
 }
+
+jacocoAndroidUnitTestReport {
+    excludes += ['**/AutoValue_*.*',
```
Comment (stephanenicolas) [inline feature-adapter-group/build.gradle]: We don't need these exclusions.

File: feature-adapter-rx/build.gradle [hunk 2]
```diff
@@ -41,3 +46,12 @@ dependencies {
   testImplementation deps.junit
   testImplementation deps.easymock
 }
+
+jacocoAndroidUnitTestReport {
+  excludes += ['**/AutoValue_*.*',
```
Comment (stephanenicolas) [inline feature-adapter-rx/build.gradle]: same

File: feature-adapter-rx/build.gradle [hunk 3]
```diff
@@ -41,3 +46,12 @@ dependencies {
...
+               '**/*JavascriptBridge.class']
+
+  csv.enabled false
+  html.enabled true
+  xml.enabled true
+}
```
Comment (stephanenicolas) [inline feature-adapter-rx/build.gradle]: missing line

File: feature-adapter/build.gradle [hunk 4]
```diff
@@ -37,3 +42,12 @@ dependencies {
...
 }
+
+jacocoAndroidUnitTestReport {
+  excludes += ['**/AutoValue_*.*',
+               '**/*JavascriptBridge.class']
+
+  csv.enabled false
+  html.enabled true
+  xml.enabled true
+}
```
Comment (stephanenicolas) [inline feature-adapter/build.gradle]: missing line

File: feature-adapter/build.gradle [hunk 5]
```diff
@@ -37,3 +42,12 @@ dependencies {
   testImplementation deps.junit
   testImplementation deps.easymock
 }
+
+jacocoAndroidUnitTestReport {
+  excludes += ['**/AutoValue_*.*',
```
Comment (stephanenicolas) [inline feature-adapter/build.gradle]: same

**Comment by stephanenicolas** [comment]
The PR looks good but I don't see the report here in the PR.

**Comment by ww2lin** [comment]
@stephanenicolas yep, just pushed to see if the tasks are running.

**Comment by stephanenicolas** [comment]
RTM. The error is just saying that codecov has no reference of code coverage on master.

**Pull request merged**